x-pack/plugins/ml/public/jobs/jobs_list/components/multi_job_actions/multi_job_actions.js

```diff
@@ -24,9 +24,10 @@ export class MultiJobActions extends Component {
 
   render() {
     const s = (this.props.selectedJobs.length > 1) ? 's' : '';
+    const jobsSelected = (this.props.selectedJobs.length > 0);
     return (
-      <div className="multi-select-actions">
-        {this.props.selectedJobs.length > 0 &&
+      <div className={`multi-select-actions${jobsSelected ? '' : '-no-display'}`}>
```

Comment:
Sounds like a good rule of thumb 👍 For this case it feels pretty safe since the style file is organizationally so close to the component it's styling.